meshapp/src/profile/profile-page.tsx

```diff
@@ -295,6 +308,7 @@ const TestComponent = (props: any) => {
       placeholder={"Test"}
       text={"Test"}
       charLimit={200}
+      handleSave={() => {return}}
```

Comment:
What is the purpose of this line? is this tied to testing? lol maybe I am dumb

Reply:
I added a 'handleSave' function parameter to the parameters in 'profile-textfield.tsx'
Because of this, each <ProfileTextField> component is expected to be passed a 'handleSave' prop (since every editable field can be saved)

As such, the test component also needs one. The empty function is just there to satisfy the parameter requirement.

Reply:
ahh makes sense. Interesting okay. Thanks for explaining to me.